### PR TITLE
Add more troubleshooting tips

### PR DESCRIPTION
**Pull request by jwmueller**
plus minor text improvements

### CODE REVIEW

File: docs/tutorials/tabular_prediction/tabular-indepth.md [hunk 1]
```diff
@@ -321,11 +321,15 @@ To reduce memory usage during training, you may try each of the following strate
...
 
-- Text fields in your table require substantial memory for N-gram featurization. To mitigate this in `fit()`, you can either: (1) add `'ignore_text'` to your `presets` list (to ignore text features), or specify:
+- Text fields in your table require substantial memory for N-gram featurization. To mitigate this in `fit()`, you can either: (1) add `'ignore_text'` to your `presets` list (to ignore text features), or (2) specify the argument:
 
-`feature_generator = autogluon.utils.tabular.features.generators.auto_ml_pipeline.AutoMLPipelineFeatureGenerator(vectorizer=CountVectorizer(min_df=30, ngram_range=(1, 3), max_features=MAX_NGRAM, dtype=np.uint8))`
+```
+feature_generator = autogluon.utils.tabular.features.generators.auto_ml_pipeline.AutoMLPipelineFeatureGenerator(vectorizer=CountVectorizer(min_df=30, ngram_range=(1, 3), max_features=MAX_NGRAM, dtype=np.uint8))
+```
+
+where `MAX_NGRAM = 1000` say (try various values under 10000 to reduce the number of N-gram features used to represent each text field) and [CountVectorizer](https://scikit-learn.org/stable/modules/generated/sklearn.feature_extraction.text.CountVectorizer.html) is imported from `sklearn.feature_extraction.text`.
```
Comment (Innixma) [inline docs/tutorials/tabular_prediction/tabular-indepth.md:330]: Consider moving the import of CountVectorizer down here so it is easier to understand?
Also consider importing AutoMLPipelineFeatureGenerator in a separate line for readability.

**Review by Innixma — APPROVED** [review]
Looks good!

**Comment by szha** [comment]
Job PR-679-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-679/1/index.html

**Comment by szha** [comment]
Job PR-679-2 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-679/2/index.html

**Comment by szha** [comment]
Job PR-679-3 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-679/3/index.html

**Pull request merged**